draft-quic-atsss-reqs.mkd

```diff
@@ -471,12 +471,28 @@ This document discusses three approaches:
 ### Single QUICv1 Tunnel with Unreliable Datagram Extension and Connection Migration
 
 
-In this case, there is one QUIC session between the UE and the ATSSS proxy.
-This QUIC session uses only one access network at a time. Thanks to the
-connection migration capability of QUIC, it is possible to support the
-switching and steering functions of ATSSS, but splitting cannot be supported. 
-
-This is not an option for ATSSSv2.
+QUIC can be used as a tunneling protocol between the UE and the ATSSS proxy. Use of the Unreliable 
+Datagram Extension avoids unnesseary delays due to local retransmissions and, more important, subsequently 
+head-of-line blocking.
+
+In this case, there is (only) one QUIC connection between the UE and the ATSSS proxy.
+And as such, that QUIC connection uses only one access network at a time. 
+However, given the connection migration capability of QUIC, the QUIC connection could be
+moved over to another access network, e.g. when indicated by the network that the currently used 
+access network would go away or capacity is limited. Depending of the current flow of traffic 
+either the UE or the ATSS proxy could initiate the connection mitragtion to start the path
```

Comment:
mitragtion -> migration
Migration is easy for UE, but more complex for ATSSS proxy. Could we write something like: 
The UE (and possibly the ATSSS proxy provided that it knows the UE addresses) could initiate the connection migration to start the path...

Reply:
Another possibility would be to have an explicit "please migrate" message from the ATSSS proxy, but I've never seen this kind of message.

Reply:
As I already mentioned, even if the proxy knows the UE's @es, this is not a connection migration as currently defined in the quic-transport spec.

Reply:
> 
> 
> Another possibility would be to have an explicit "please migrate" message from the ATSSS proxy, but I've never seen this kind of message.

If there is such need, the UE will use a distinct IP address so that the connection is migrated. I don't see the need for an extra signal. No?

Reply:
Removed it. Should we open an issue to further discuss later?